luxonis_train/config/predefined_models/anomaly_detection_model.py

```diff
@@ -103,7 +106,7 @@ def metrics(self) -> list[MetricModuleConfig]:
             MetricModuleConfig(
                 name="JaccardIndex",
                 attached_to=f"{self.task_name}-DiscSubNetHead",
-                params={"num_classes": 2, "task": "multiclass"},
+                params={"num_classes": 1, "task": "binary"},
```

Comment:
I think this should still be `"multiclass"`. It is technically binary, but we're explicitly using 2 classes for background and anomaly (line 101 in `perlin_loader.py`)